src/visualization.py

```diff
@@ -1,18 +1,17 @@
 import numpy as np
 import matplotlib.patches as patches
 import matplotlib.pyplot as plt
+import cv2
 
 
 def show_frame(frame, bbox, fig_n):
-    fig = plt.figure(fig_n)
-    ax = fig.add_subplot(111)
-    r = patches.Rectangle((bbox[0],bbox[1]), bbox[2], bbox[3], linewidth=2, edgecolor='r', fill=False)
-    ax.imshow(np.uint8(frame))
-    ax.add_patch(r)
-    plt.ion()
-    plt.show()
-    plt.pause(0.001)
-    plt.clf()
+    #Adjust Color Channels of frame
+    frame_adjusted = np.ndarray(shape=(720,1280,3), dtype=np.dtype(np.uint8))
```

Comment:
I think this line should be 

`frame_adjusted = np.ndarray(shape=np.shape(frame), dtype=np.dtype(np.uint8))`